2024/FOSDEM/osf_on_amd_5th.md

```diff
@@ -0,0 +1,287 @@
+    * Scalable with any host firmware interface/framework
+    * Proof of Concept code for Genoa-based (AMD EPYC 9004 processors)
+      reference board Onyx available on
+      [GitHub](https://github.com/openSIL/openSIL)
+- [coreboot source also available and merged to upstream
+  repository](https://review.coreboot.org/q/topic:%22amd_genoa_opensil%22)
+- UEFI EDK2-based PoC code also available on GitHub:
+    * [opensil-uefi-interface](https://github.com/openSIL/opensil-uefi-interface)
+    * [EDK2 Platforms](https://github.com/openSIL/EDKII-Platform)
+- More about OpenSIL:
+    * [OCP Global Summit 2023](https://www.youtube.com/watch?v=nSVKKMkcIPE)
+    * [OSFC 2023](https://vimeo.com/878219919)
+
+---
+
+# AMD OpenSIL future
+
+.left-column55[
+.center[.image-90[![](/img/opensil_github.png)]]
+]
+
+.right-column45[
+- The OpenSIL is constantly developed and improved by AMD and partners
+- Planned to go **production in 2026** with the 6th generation of AMD EPYC
+  processors
+- AMD plans to cover the **client segment with OpenSIL support** too (Ryzen
+  desktop and mobile processors)
+]
+
+---
+
+# PC Engines platforms - yet another chance?
+
+- Last year we announced the end of PC Engines' sponsorship of open-source
+  firmware
+- We tried to gather community interested in the open-source firmware for apu2
+  board and launch a subscription model, however the response was very little
+  and we didn't succeed
+- This year we see another chance to revive the project under
+  [Dasharo](https://dasharo.com) brand
+
+.center.image-40[![](/img/pcengines_logo.png)]
+.center[
+.image-30[![](/img/apu4d4.png)]
+.image-20[![](/img/dasharo-sygnet.png)]
+]
+
+???
+
+---
+
+# Dasharo
+
+- In [Dasharo](https://dasharo.com) we carefully select hardware platforms,
+  which are quite popular
+    * Of course most such boards will be community-driven effort and the work
+    would be done in free time thus the releases may not be that frequent
+- Value offered by Dasharo:
+    * [high quality documentation](https://docs.dasharo.com) explaining:
+      initial deployment, firmware update and recovery
+    * hardware and software tools available
+    * active and helpful community on Matrix
+    * binary releases
+    * transparent validation
+- Sign up to [Dasharo
+  newsletter](https://newsletter.3mdeb.com/subscription/wwL90UkXP) to get up
+  to date information about supported platforms and the their status
+
+???
+
+Briefly explain what Dasharo is
+
+---
+
+# Dasharo
+
+- We have also introduced subscription model for selected platforms to help
+  gather funds for development
+    * in return the releases for subscribers are more frequent and available
+      with subscriber's credentials
+- There are also beta testers programs for selected platform, like [NovaCustom
+  laptops](https://novacustom.com/forum/d/15-call-for-beta-testers)
+    * Beta testers are given access to pre-release firmware to assist with
+      testing features and reporting bugs
```

Comment:
There is no word about the NLNet grant for AMD TrenchBoot; I know Maciej and Jag will talk about that in detail, but one bullet point should be in this presentation for those who will not join TrenchBoot.

Reply:
Sure, I can briefly mention it.

Reply:
Added a slide: https://github.com/3mdeb/conferences/pull/20/commits/71382d62a8fcec201d0db24cc3ab111f4d499127#diff-a36529ceeba71601b1edbcceb8639bd8b743d579e8321eff59d469b53e6ad1d0R292-R298